### PR TITLE
Move utils to heron instance

### PR DESCRIPTION
**Pull request by aahmed-se**
This change to remove heron-api dependency for heron-common

### CODE REVIEW

File: heron/common/tests/python/BUILD [hunk 1]
```diff
@@ -5,7 +5,7 @@ pex_library(
     name = "pytest-py",
     srcs = ["mock_protobuf.py"],
     deps = [
-      "//heron/common/src/python:common-py",
+      "//heron/instance/src/python:instance-py",
```
Comment (srkukarni) [inline heron/common/tests/python/BUILD]: Is this correct?

Comment (aahmed-se) [inline heron/common/tests/python/BUILD]: fixed

File: heron/instance/src/python/network/st_stmgr_client.py [hunk 2]
```diff
@@ -17,7 +17,8 @@
 
 from heron.common.src.python.config import system_config
 from heron.common.src.python.utils.log import Log
-from heron.common.src.python.network import HeronClient, StatusCode
+from heron.instance.src.python.network.heron_client import HeronClient
+from heron.common.src.python.network import StatusCode
```
Comment (kramasamy) [inline heron/instance/src/python/network/st_stmgr_client.py]: move one line up

File: heron/instance/tests/python/network/mock_generator.py [hunk 3]
```diff
@@ -15,10 +15,10 @@
 # pylint : disable=missing-docstring
 from heron.common.src.python.basics import EventLooper
 from heron.common.src.python.network import SocketOptions
-from heron.common.src.python.utils.misc import HeronCommunicator
+from heron.instance.src.python.utils.misc import HeronCommunicator
 from heron.instance.src.python.network import SingleThreadStmgrClient, MetricsManagerClient
 import heron.common.src.python.system_constants as constants
```
Comment (kramasamy) [inline heron/instance/tests/python/network/mock_generator.py]: aggregate all common and instance references?

File: heron/instance/tests/python/utils/BUILD [hunk 4]
```diff
@@ -16,8 +16,8 @@ pex_test(
     name = "communicator_unittest",
     srcs = ["communicator_unittest.py"],
     deps = [
-        "//heron/common/tests/python:pytest-py",
-        "//heron/common/tests/python/utils:common-utils-mock"
+        "//heron/instance/tests/python:pytest-py",
```
Comment (kramasamy) [inline heron/instance/tests/python/utils/BUILD]: can you change the target pytest-py to something like instance-tests-py

File: heron/instance/src/python/network/st_stmgr_client.py (hunk 2 above)
Comment (aahmed-se) [inline heron/instance/src/python/network/st_stmgr_client.py]: done

File: heron/instance/tests/python/network/mock_generator.py (hunk 3 above)
Comment (aahmed-se) [inline heron/instance/tests/python/network/mock_generator.py]: done

File: heron/instance/tests/python/utils/BUILD (hunk 4 above)
Comment (aahmed-se) [inline heron/instance/tests/python/utils/BUILD]: done

File: heron/tools/cli/src/python/BUILD [hunk 5]
```diff
@@ -10,6 +10,7 @@ pex_library(
     deps = [
         "//heron/common/src/python:common-py",
         "//heron/tools/common/src/python:common-py",
+        "//heron/instance/src/python:instance-py",
```
Comment (srkukarni) [inline heron/tools/cli/src/python/BUILD]: pex loader should go back to common. That way tools will not depend on instance

Comment (aahmed-se) [inline heron/tools/cli/src/python/BUILD]: done

**Pull request merged**